Source/Fuse.Controls.TimePicker/TimePicker.ScriptClass.uno

```diff
@@ -0,0 +1,91 @@
+using Uno;
+using Uno.Text;
+using Uno.UX;
+
+using Fuse;
+using Fuse.Controls.Native;
+using Fuse.Scripting;
+
+namespace Fuse.Controls
+{
+	public partial class TimePickerBase
+	{
+		class ValueProperty : Property<DateTime>
+		{
+			readonly TimePickerBase _tp;
+			public override PropertyObject Object { get { return _tp; } }
+			public override bool SupportsOriginSetter { get { return false; } }
+			public override DateTime Get(PropertyObject obj)
+			{
+				DateTime date = DateTime.UtcNow;
+				lock(_tp)
+					date = _tp._outValue;
+				return date;
+			}
+			public override void Set(PropertyObject obj, DateTime value, IPropertyListener origin)
+			{
+				lock(_tp)
+					_tp._inValue = value;
+				UpdateManager.PostAction(_tp.UpdateValue);
+			}
+			public ValueProperty(TimePickerBase timePicker) : base(TimePickerBase._valueName) { _tp = timePicker; }
+		}
+
+		class Is24HourViewProperty : Property<bool>
+		{
+			readonly TimePickerBase _tp;
+			public override PropertyObject Object { get { return _tp; } }
+			public override bool SupportsOriginSetter { get { return false; } }
+			public override bool Get(PropertyObject obj)
+			{
+				bool is24HourView = false;
+				lock(_tp)
+					is24HourView = _tp._outIs24HourView;
+				return is24HourView;
+			}
+			public override void Set(PropertyObject obj, bool value, IPropertyListener origin)
+			{
+				lock(_tp)
+					_tp._inIs24HourView = value;
+				UpdateManager.PostAction(_tp.UpdateIs24HourView);
+			}
+			public Is24HourViewProperty(TimePickerBase timePicker) : base(TimePickerBase._is24HourViewName) { _tp = timePicker; }
+		}
+
+		static TimePickerBase()
+		{
+			ScriptClass.Register(typeof(TimePickerBase),
+				new ScriptProperty<TimePickerBase, DateTime>("value", GetValueProperty),
+				new ScriptProperty<TimePickerBase, bool>("is24HourView", GetIs24HourViewProperty));
```

Comment:
Same with this, we usually don't expose config variables on the JS interface as there is no need. The user can do `<TimePicker Is24HourView="{jsBool}">` if they want.